ani.voskanyan/repo/Lesson_6/exercise_12/exercise_12.py

```diff
@@ -0,0 +1,20 @@
+#!usr/bin/env python3
+
+negative_numbers = []
+positive_numbers = []
+zero_list = []
+
+number_input = input("Enter a number: ")
+
+while number_input != '':
+    if int(number_input) < 0:
+        negative_numbers.append(number_input)
+        number_input = input("Enter a number: ")
+    elif int(number_input) > 0:
+        positive_numbers.append(number_input)
+        number_input = input("Enter a number: ")
+    else:
+        zero_list.append(number_input)
+        number_input = input("Enter a number: ")
+else:
+    print(negative_numbers + zero_list + positive_numbers)

```

Comment:
RE: The output should be ordered.

Reply:
But exercise requirements is "...Within each group the numbers should be displayed in the same order that they were entered by the user. For example, if the user enters the values 3, -4, 1, 0, -1, 0, and -2 then your program should output the values -4, -1, -2, 0, 0, 3, and 1." Here the output is not ordered :/

Reply:
Yes, you are right. Removing my comment